3 reports.sql

```diff
@@ -0,0 +1,19 @@
+use RealEstateDb
+
+--1) I want a report of all the houses sorted by the block and then by the town/city
```

Comment:
It seems like from this question that it is important to split up the house address into 2 columns: House number and Street

Reply:
Okay, great